pathfinder/paizo/player_companion/alchemy_manual/alchemy_manual.pcc

```diff
@@ -0,0 +1,71 @@
+SOURCESHORT:AM
+SOURCEWEB:http://paizo.com/products/btpy90bj
+SOURCEDATE:2014-04
+ISOGL:YES
+
+COVER:alchemy_manual.jpeg
+LOGO:@/publisher_logos/Paizo_Publishing.png
+URL:WEBSITE|http://www.paizo.com/|Visit Paizo Publishing!
+URL:WEBSITE|http://www.paizo.com/communityuse|Paizo's Community Use Policy.
+URL:Paizo Publishing|http://paizo.com/products/btpy90bj|Buy this publication from Paizo Publishing!
+DESC:Harness the powers of alchemy with Pathfinder Player Companion: Alchemy Manual! Along with nearly 100 never-before-seen alchemical items, this volume features a brand-new rules subsystem for creating potent alchemical items on the fly using a variety of alchemical reagents and alchemical recipes. Whether you�re brewing the magical dwarven ales of the Five Kings Mountains or honing the deadly secrets of the Daggermark Poisoners� Guild, you�ll discover all the tools you need inside the Alchemy Manual!
+
+INFOTEXT:This dataset uses trademarks and/or copyrights owned by Paizo Inc., which are used under Paizo's Community Use Policy. We are expressly prohibited from charging you to use or access this content. This dataset is not published, endorsed, or specifically approved by Paizo Publishing. For more information about Paizo's Community Use Policy, please visit paizo.com/communityuse. For more information about Paizo Publishing and Paizo products, please visit paizo.com.
+
+COPYRIGHT:Open Game License v 1.0a Copyright 2000,Wizards of the Coast, Inc.
+COPYRIGHT:System Reference Document Copyright 2000, Wizards of the Coast, Inc; Authors: Jonathan Tweet, Monte Cook, and Skip Williams, based on material by E. Gary Gygax and Dave Arneson.
+
+COPYRIGHT:Kobold Quarterly, Issue 7 � 2008, Open Design LLC, www.koboldquarterly.com; Authors: John Baichtal, Wolfgang Baur, Ross Byers, Matthew Cicci, John Flemming, Jeremy Jones, Derek Kagemann, Phillip Larwood, Richard Pett, and Stan!
+COPYRIGHT:Ooze, Magma from the Tome of Horrors Complete � 2011, Necromancer Games, Inc., published and distributed by Frog God Games; Author: Scott Greene.
+COPYRIGHT:Slithering Tracker from the Tome of Horrors Complete � 2011, Necromancer Games, Inc., published and distributed by Frog God Games; Author: Scott Greene, based on original material by Gary Gygax.
+COPYRIGHT:Pathfinder Player Companion: Alchemy Manual � 2014, Paizo Inc.; Authors: Jason Nelson, Patrick Renie, and David N. Ross.
+
+COPYRIGHT:PCGen dataset conversion for "Pathfinder Player Companion: Alchemy Manual" Copyright 2016, PCGen Data Team (including, but not limited to Vincent Hatakeyama)
+
+# Do not remove the EXTRAFILE comment.
+#EXTRAFILE:OGL.txt
+
+# Data files
+
+#ABILITY:aa_abilities.lst
+#ABILITY:aa_abilities_variant.lst
+#ABILITYCATEGORY:aa_abilitycategories.lst
+#CLASS:aa_classes.lst
+#DOMAIN:aa_domains.lst
+EQUIPMENT:aa_equip.lst
+#FEAT:aa_feats.lst
+#RACE:aa_races.lst
+#SKILL:aa_skills.lst
+#SPELL:aa_spells.lst
+#TEMPLATE:aa_templates.lst
+
+# ENTRY DATE: 
+# LST MONKEY: 
+# LICENSE MONKEY:
```

Comment:
Inconsistent naming. AA instead of am
File names at least one file incorrect
Please notify when correction pushed.